service/gcs/main.go

```diff
@@ -17,6 +14,7 @@ import (
 	"github.com/Microsoft/opengcs/service/gcs/core/gcs"
 	"github.com/Microsoft/opengcs/service/gcs/runtime/runc"
 	"github.com/Microsoft/opengcs/service/gcs/transport"
+	"github.com/Microsoft/opengcs/service/libs/commonutils"
```

Comment:
I know the code has these but this is *legacy*. Golang should use `internal/<pkg>` or `pkg/<pkg>`. Since this is internal lets call it `internal/debug` or something.